src/resolve-config.ts

```diff
@@ -88,3 +90,10 @@ export const getDefinedInput = (name: string): string | undefined => {
 
   return input !== '' ? input : undefined
 }
+
+export const getDefinedBoolean = (name: string): boolean | undefined => {
+  if (!getDefinedInput(name)) {
+    return undefined
+  }
+  return core.getBooleanInput(name)
```

Comment:
`core.getBooleanInput()` [will throw an error](https://github.com/actions/toolkit/pull/725/files#diff-f63fb32fca85d8e177d6400ce078818a4815b80ac7a3319b60d3507354890992R110) if the property isn't valid YAML, i.e. not `true|false`, `True|False` or `TRUE|FALSE`. So I think it's a good behavior 👍